lourdes_villca/tic_tac_toe/src/main/tictactoe/Main.java

```diff
@@ -0,0 +1,75 @@
+package tictactoe;
+
+import java.util.Scanner;
+
+public class Main {
+    /**
+     * Main method to run the game.
+     *
+     * @param args
+     */
+    public static void main(String[] args) {
+
+        Board board = new Board();
+        System.out.println("STARTING --- TIC TAC TOE GAME");
+        board.printBoard();
+        Player player1 = new Player("A", "X");
+        Player player2 = new Player("B", "O");
+        System.out.println("\nPlayer: " + player1.getNickname() + " - Symbol: " + player1.getSymbol());
+        System.out.println("Player: " + player2.getNickname() + " - Symbol: " + player2.getSymbol());
+
+        while (!board.checkWinner()) {
+            executePlayerMove(player1, board);
+            executePlayerMove(player2, board);
+        }
+    }
+
+    /**
+     * Method that execute the Player move.
```

Comment:
It reads that this method belongs to Player, what does the Play do ? Move ?

Reply:
Done, method moved to Board Class